spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -0,0 +1,58 @@
+describe ServiceTemplateTransformationPlanTask do
+  describe '.base_model' do
+    it { expect(described_class.base_model).to eq(ServiceTemplateTransformationPlanTask) }
+  end
+
+  describe '#after_request_task_create' do
+    it 'overrides #after_request_task_create' do
+      expect(described_class.instance_methods(false)).to include(:after_request_task_create)
```

Comment:
Same here.  We should be testing the result of the method and not just the existence of the override.